plugin/PDFDocumentPage.qml

```diff
@@ -27,6 +27,10 @@ import "PDFStorage.js" as PDFStorage
 DocumentPage {
     id: base
 
+    property alias document: pdfDocument
+    property alias placeHolder: placeHolderLoader
+    property alias toolbar: toolbar
+
```

Comment:
Commonly not fond of adding very much code used only for testing, but can have some. Just thinking it could be marked for such usage so it's not accidentally removed as cleanup. Maybe could also have all testability properties of the file grouped together?